portfolio/src/main/webapp/index.html

```diff
@@ -9,11 +9,71 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <p>This is Steven's portfolio.</p>
```

Comment:
This PR seems to include changes from a prior PR. Please merge prior PR and rebase with master to exclude these changes.